company/how-to-guides-for-staff/how-to-setup-a-1-1-channel.md

```diff
@@ -2,15 +2,15 @@
 description: 50% complete.
 ---
 
-# How to setup a 1-1 channel
+# How to set up a 1-1 channel
 
-## Getting Started
+## Getting started
 
-1. **Create a Google Doc.** Name the file `[INITIALS_OF_FIRST_PERSON]-[INITIALS_OF_SECOND_PERSON] 1-1`. 
-   1. For example. if Bob Adrian Jones is meeting with Jen Tai Yin Lee, the title of the file would be `BAJ-JTL 1-1.` ****
+1. **Create a Google Doc** and name the file `[INITIALS_OF_FIRST_PERSON]-[INITIALS_OF_SECOND_PERSON] 1-1`.
+   1. For example. if Bob Adrian Jones is meeting with Jen Tai Yin Lee, the title of the file would be `BAJ-JTL 1-1.`
    2. Use three letter initials when possible since there may be people with the same initials. 
-   3. The manager, or the more senior person should be listed last. If seniority is the same, the person with the last name that starts with a letter earliest in the alphabet should be listed first. 
-2. **Share the file** with the other person 
-3. **Add a link to the Google Doc** at the top of the Direct Message channel with the other person click on the downward arrow and choose "Edit Header" and paste the link. You can do this by editing the header of the DM channel 
-4. **Use the file for 1-1s.** When it's time for 1-1s you can use the Google Doc to post material in advance and to write notes of your discussions. 
-
+   3. The manager, or the more senior person, should be listed last. If seniority is the same, the person with the last name that starts with a letter earliest in the alphabet should be listed first.
```

Comment:
I'm wondering if this step is necessary, but we can leave it, 0/5

Reply:
Unless that specific format is required for adding to a system that requires precise formatting (e.g. an auditing system), I'm happy to remove it.